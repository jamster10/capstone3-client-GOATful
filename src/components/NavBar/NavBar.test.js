import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer'
import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';


import NavBar from './NavBar';

import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'

configure({ adapter: new Adapter() });

it('NavBar renders without crashing', () => {
    const wrapper = shallow(<NavBar />)
    expect(toJson(wrapper)).toMatchSnapshot()
});
