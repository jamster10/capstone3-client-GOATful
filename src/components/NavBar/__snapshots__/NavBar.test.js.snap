// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavBar renders without crashing 1`] = `
<header>
  <div
    className="title-container"
  >
    <Link
      className="root-link"
      replace={false}
      to="/"
    >
      <Logo
        className="logo"
      />
      <h2
        className="app-title"
      >
        Laconic
      </h2>
    </Link>
  </div>
  <nav>
    <Link
      replace={false}
      to="/login"
    >
      Login
    </Link>
     
  </nav>
</header>
`;
